### ISSUE TITLE
Child jobs of ParallelMaster

### ISSUE THREAD

**Issue by Leimeroth** [pyiron/pyiron_atomistics#212]
**Original Issue**
As discussed in #197 ParallelMaster jobs don't know about their child jobs when submitting them to a remote queue. I decided to open another issue for this since it is completely unrelated to the problem with ElasticTensor jobs.
I thought there must have been some change to the Class since it used to work before, but maybe I was wrong with that.
After revisiting some old notebooks I noticed that child jobs are created and submitted individually when a queue is assigned to the ref job, but not to the master job, which is probably what I always did. If a queue is specified for the master job it get's submitted and child jobs are created on the remote cluster, so their IDs are lost when fetching them back to the local system.

As suggested by @liamhuber I had a look at the _run_if_master_modal_child_non_modal function, but as far as I can tell all the Master run functions are called after the job is transferred to the cluster so any change in them won't have an effect in the local database, so I am somewhat lost where this can be fixed without messing up the installation of everyone running a different setup. 

**Update**
pyiron/pyiron_base#329 fixes the problem when the master job finishes. The issue persists if the master job itself fails. It is unclear yet what happens if a child job fails.

**Comment by jan-janssen** [comment]
Can you try https://github.com/pyiron/pyiron_base/pull/329 ?

**Comment by Leimeroth** [comment]
This solves the problem when the job is successful. It does not restore the child ids for the failing ElasticTensor jobs with Vasp (#197).

**Comment by jan-janssen** [comment]
I agree that the elastic tensor job has to be adjusted to work with the VASP pressure output of the VASP pressure has to be adjusted to return an 3x3 tensor, but I guess that is a separate issue, which is discussed in https://github.com/pyiron/pyiron_atomistics/issues/197 .

**Comment by jan-janssen** [comment]
@Leimeroth Is this issue fixed?
